packages/ens-controller/package.json

```diff
@@ -65,7 +65,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^21.0.0"
+    "@metamask/network-controller": "^22.0.0"
```

Comment:
Peer dependency major version bumps are breaking changes. I suppose not releasing the dependents isn't a problem, but adding a changelog section like the following to the PR description would be nice -- just to make life easier for whoever ends up releasing the next versions of network-controller dependent packages and needs to go digging to find where this peerDep bump happened.

```md
## Changelog

### Changed

- **BREAKING:** Bump `@metamask/network-controller` peer dependency from `^21.0.0` to `^22.0.0` ([#4841](https://github.com/MetaMask/core/pull/4841))
```

Reply:
And that would go in the ## [Unreleased] section of each changelog?

Reply:
Oh I was thinking of the PR description, but the unreleased section would be even better. That would definitely be very helpful to the next person.